app/controllers/check_controller.rb

```diff
@@ -1,4 +1,4 @@
-class CheckController < ApplicationController
+class CheckController < APIController
```

Comment:
This should skip the `authenticate` method of APIController, because it needs to be accessible without HTTP AUTH (it's used by the load balancers to check status, and could cause the containers to be endlessly restarted)